main.cognitiveservices.tf

```diff
@@ -0,0 +1,25 @@
+resource "azapi_resource" "aiservice" {
+  count = var.kind == "hub" ? 1 : 0
+
+  type = "Microsoft.CognitiveServices/accounts@2024-04-01-preview"
+  body = jsonencode({
+    properties = {
+      publicNetworkAccess = var.is_private ? "Disabled" : "Enabled"
+      apiProperties = {
+        statisticsEnabled = false
```

Comment:
should this always be disabled?

Reply:
perhaps make the default var is_private=true